src/compiler/stack.ts

```diff
@@ -26,9 +20,17 @@ function stacking(encoding, mdef, stack) {
     });
   }
 
+  interface StackTransform {
```

Comment:
Later. For now I mostly added `any` types and this is one of the few places where I didn't do it.